src/main/kotlin/org/jetbrains/research/tasktracker/handler/ide/SettingsHandler.kt

```diff
@@ -20,7 +30,54 @@ class SettingsHandler(override val config: SettingsConfig) : BaseHandler {
         SettingMode.OFF -> settings.AUTO_POPUP_COMPLETION_LOOKUP = false
     }
 
+    private fun setupZenMode(project: Project) {
+        val isZenModeEnabled = ToggleZenModeAction.isZenModeEnabled(project)
+        when (config.enableZenMode) {
+            SettingMode.DEFAULT -> {}
+            SettingMode.ON -> if (!isZenModeEnabled) changeZenModeState()
+            SettingMode.OFF -> if (isZenModeEnabled) changeZenModeState()
+        }
+    }
+
+    /**
+     * Turns on Zen mode if it's off and turns it off if it's on.
+     */
+    private fun changeZenModeState() {
+        val actionManager = ActionManager.getInstance()
+        ApplicationManager.getApplication().invokeLater {
+            actionManager.getAction(zenModeActionId)?.let { action ->
+                actionManager.tryToExecute(action, null, null, null, true)
+            }
+        }
+    }
+
+    private fun setupTheme() {
+        when (config.theme) {
+            Theme.DEFAULT -> {}
+            Theme.LIGHT -> {
+                val lightTheme = lafManager.installedLookAndFeels.find { it.name == light }
```

Comment:
I'd suggest to move 
```kotlin
lafManager.installedLookAndFeels.find { it.name == theme }
                    ?: error("theme_name theme must exist")
```
into another function, to be easier to add more themes (e.g. a custom one) here